src/helpers/getDeckAdvice/spec.js

```diff
@@ -54,184 +54,299 @@ const getCards = id =>
+    disabled: true,
+  },
+  {
+    advice: 'LACK_OF_AOE',
+    deck: '1n11n21n31n671n71n101n201i141n651n271n291i28',
+    label: 'should handle lack of aoe (Bladestorm)',
+    disabled: true,
+  },
+  {
+    advice: 'LACK_OF_AOE',
+    deck: '1n11n21n31n671n71n101n201i141n651n271n361i28',
+    label: 'should handle lack of aoe (Voidsurgers)',
+    disabled: true,
+  },
+  {
+    advice: 'LACK_OF_AOE',
+    deck: '1n11n21n31n671n71n101n201i141n651n271i281n44',
+    label: 'should handle lack of aoe (Needle Blast)',
+    disabled: true,
+  },
+  {
+    advice: 'LACK_OF_AOE',
+    deck: '1n11n21n31n671n71n101n201i141n651n271i281n45',
+    label: 'should handle lack of aoe (Powder Tower)',
+    disabled: true,
+  },
+  {
+    advice: 'LACK_OF_AOE',
+    deck: '1n11n21n31n671n71n101n201i141n651n271i281n55',
+    label: 'should handle lack of aoe (Joust Champions)',
+    disabled: true,
+  },
+  {
+    advice: 'LACK_OF_AOE',
+    deck: '1n11n21n31n671n71n101n201i141n651n271i281n57',
+    label: 'should handle lack of aoe (Crazy Bombers)',
+    disabled: true,
+  },
+  {
+    advice: 'LACK_OF_AOE',
+    deck: '1n11n21n31n671n71n101n201i141n651n271i281n58',
+    label: 'should handle lack of aoe (Siren of the Seas)',
+    disabled: true,
+  },
+  {
+    advice: 'LACK_OF_AOE',
+    deck: '5n15n25n33n632n673n72n102n204i142n653n272i28',
+    label: 'should handle lack of aoe (Unhealthy Hysteria)',
+    disabled: true,
+  },
+  {
+    advice: 'LACK_OF_AOE',
+    deck: '1n11n21n31n671n71n101n731n201i141n651n271i28',
+    label: 'should handle lack of aoe (Trekking Aldermen)',
+    disabled: true,
+  },
+  {
+    advice: 'LACK_OF_AOE',
+    deck: '1n11n21n31n41n621n151w71n241w81w161n381w15',
+    label: 'should handle lack of aoe (Broken Earth Drake)',
+    disabled: true,
+  },
+  {
+    advice: 'LACK_OF_AOE',
+    deck: '1n11n21n31n671n71n101n201i141n651n271i181i28',
+    label: 'should handle lack of aoe (Flaming Stream)',
+    disabled: true,
+  },
+  {
+    advice: 'LACK_OF_AOE',
+    deck: '1n11n21n31n671n71n101n201i141n651n271i201i28',
+    label: 'should handle lack of aoe (Windmakers)',
+    disabled: true,
+  },
+  {
+    advice: 'LACK_OF_AOE',
+    deck: '1n21f41f31n31n661f271n591n211n751n641f161n76',
+    label: 'should handle lack of aoe (Toxic Sacrifice)',
+    disabled: true,
+  },
+  {
+    advice: 'LACK_OF_AOE',
+    deck: '1n21f31n31n661f51f271n591n211n751n641f161n76',
+    label: 'should handle lack of aoe (Crimson Sentry)',
+    disabled: true,
+  },
+  {
+    advice: 'LACK_OF_AOE',
+    deck: '1n21f31n31n661f271n591n211n751n641f141f161n76',
+    label: 'should handle lack of aoe (Witches of the Wild)',
+    disabled: true,
+  },
+  {
+    advice: 'LACK_OF_AOE',
+    deck: '1n631n141s251n191n91s121n241n261s151s191n461n48',
+    label: 'should hand lack of aoe (Dark Harvest)',
+    disabled: true,
+  },
 ]
 
 describe('The `getDeckAdvice` helper', () => {
+  TESTS.forEach(adv => {
```

Comment:
The object in the loop is not an advice, it’s a test case. So maybe naming it `test` would be better?